src/Package/Impl/ExportDialog/ExportImageDialog.xaml.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Windows;
+using System.Windows.Controls;
+using System.ComponentModel;
+using Microsoft.Common.Core;
+using Microsoft.VisualStudio.R.Package.Wpf;
+
+namespace Microsoft.VisualStudio.R.Package.ExportDialog {
+    public partial class ExportImageDialog : PlatformDialogWindow, INotifyPropertyChanged {
```

Comment:
This is usually a bad idea to mix `INotifyPropertyChanged` and `DependencyObject` in one class.